libs/search/src/lib/search/search.component.html

```diff
@@ -1,9 +1,18 @@
 <p>Search:</p>
 
 <ng-workspace05-nx-input></ng-workspace05-nx-input>
-<label>
-  <input type="checkbox" [(ngModel)]="isGlobalSearch" /> Global search?
-</label>
-<ng-workspace05-nx-button>Submit</ng-workspace05-nx-button>
+<br>
+<br>
+<div>
+	<ng-workspace05-nx-checkbox [checked]="isGlobalSearch" (checkedChange)="isGlobalSearch = !isGlobalSearch">Label for checkbox</ng-workspace05-nx-checkbox>
```

Comment:
You can improve the usage of `checkbox` component by emitting checked value in `checkedChange` event - then you can just replace here the usage like so:
```html
<ng-workspace05-nx-checkbox [(checked)]="isGlobalSearch">Label for checkbox</ng-workspace05-nx-checkbox>
```